app/src/main/java/org/sopt/at/HomeActivity.kt

```diff
@@ -113,10 +113,10 @@ fun CategoryTab() {
         categories.forEachIndexed { index, title ->
             Text(
                 text = title,
-                modifier = Modifier
-                    .clickable { selectedIndex = index },
-                fontWeight = if (selectedIndex == index) FontWeight.Bold else FontWeight.Normal,
-                color = if (selectedIndex == index) Color.White else Color.Gray
+                modifier = Modifier.clickable { selectedIndex = index },
+                color = if (selectedIndex == index) Color.White else Color.Gray,
+                fontSize = 18.sp,
+                fontWeight = if (selectedIndex == index) FontWeight.Bold else FontWeight.Normal
```

Comment:
color랑 typo 설정했는데 왜 안씀 !

```suggestion
                Text(
                    text = stringResource(R.string.home_today_top_20_title),
                    style = typography.title1.b20,
                    color = colors.white,
                    modifier = Modifier.padding(start = 20.dp, top = 15.dp)
                )
```
이건 제 코드인데 참고하시라고 남겨봅니다 ~ 
설정한 color와 typo를 사용하면 아주 깔쌈해져요